{{cookiecutter.github_repository}}/provisioner/roles/project_data/tasks/main.yml

```diff
@@ -18,7 +18,7 @@
 - name: Install packages based on package.json.
   npm: path={{ project_path }} global=no
   become: true
-  when: sass_with_django_compressor
+  when: webpack
```

Comment:
Maybe be more specific as `webpack_enabled`, but don't know the exact wording used